git-functions.sh

```diff
@@ -9,10 +9,20 @@ function git.rebase.interactive {
     fi
 }
 
+alias gri=git.rebase.interactive
 
-# create a working branch, similar to `git worktree`
-# but works by creating a local clone 
+function git.wipbranch {
+  if [ -d .git ]; then
+      if [ $1 ]; then 
+          git checkout -b ${GH_USER_PREFIX}${1}
+      fi
+  fi
+}
 
+# create a working branch, similar to `git worktree`
+# but works by creating a local clone. 
+# If env variable 'GH_USER_PREFIX' is set, it's used as a prefix for the branch, 
+# eg: GH_USER_PREFIX=wip- results in wip-{chosen-branch-name}
```

Comment:
create a branches with a prefix, for example, I always prefix my branches with `octonato`.